components/bl-calendarHeatmap-component/component.json

```diff
@@ -18,7 +18,8 @@
       "handlerId": "calendarDataLogic",
       "handlerLabel": "Calendar Data Logic",
       "handlerDescription": "This is a handler for the logic to determine the data array of the calendar(date, count).",
-      "dataBinding": true
+      "dataBinding": true,
+      "settingTooltip": "Allows you to specifies an array of date and count for calendar. Signature of Calendar Data: {date: string, count: number}"
```

Comment:
`Allows to specify an array of date and count for the calendar. Signature of Calendar Data: {date: string, count: number}`

Remove `you` everywhere.

Reply:
Use https://www.reverso.net/ + grammarly

That helps to avoid most common mistakes